### ISSUE TITLE
Possible new feature

### ISSUE THREAD

**Issue by termitaklk** [fallenstardust/YGOMobile-cn-ko-en#167]
Hello good afternoon, a question. Would it be possible to implement something similar to edopro that uses the whitelist property to create formats in the game?

With the whitelist you can define the list of banned, limited, semi-limited and allowed cards. That is, if you specify a total of 5 thousand cards in the banlist, only these will appear in the deck editor when selecting this format. This would be very useful to be able to add alternative formats such as goat, edison, hat and tengu.

**Comment by fallenstardust** [comment]
users can import lflist.conf of their own rule. just open lflist.conf files through YGOMobile

**Comment by VaHReX** [comment]
Yes, but the whitelist works the opposite way to the banlist, while the banlist allows you to limit cards by adding them to the iflist.conf file, the whitelist excludes cards that are not in that file, so if for example I add 40 cards, only these cards will be displayed in the game. This method is used in alternative formats, such as EDISON, where there is a limited number of cards since the banlist dates back to 2010, the whitelist in this case would contain the cards that were released up to that date, and it is convenient for duelists who want to play that format since they would not see the cards that came out after 2010.

This is an example of the EDISON format whitelist:

[2010.03 Edison.lflist.zip](https://github.com/user-attachments/files/17506646/2010.03.Edison.lflist.zip)
